data/yaml/venues/finnlpkdfeconlp.yaml

```diff
@@ -0,0 +1,4 @@
+acronym: FinNLPKDFECONLP
```

Comment:
Joint workshops do not receive new venue identifiers. It should be associated with each venue separately and then linked to both.

Reply:
Done.